src/components/NewMovie/NewMovie.js

```diff
@@ -1,13 +1,137 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import './NewMovie.scss';
 
 export class NewMovie extends Component {
-  state = {};
+  state = {
+    title: '',
+    description: '',
+    imgUrl: '',
+    imdbUrl: '',
+    imdbId: '',
+  };
+
+  inputHandler = (event) => {
+    const { name, value } = event.target;
+
+    this.setState({ [name]: value });
+  }
 
   render() {
+    const { title, description, imgUrl, imdbUrl, imdbId } = this.state;
+
+    const { onAdd } = this.props;
+
     return (
-      <form>
-        Put the form here
+      <form
+        onSubmit={(event) => {
+          event.preventDefault();
+          const newMovie = {
+            title,
+            description,
+            imgUrl,
+            imdbUrl,
+            imdbId,
+          };
+
+          this.setState({
+            title: '',
+            description: '',
+            imgUrl: '',
+            imdbUrl: '',
+            imdbId: '',
+          });
+
+          onAdd(newMovie);
+        }}
+      >
+
+        <label htmlFor="title">
+          Title:&nbsp;
+          <input
+            required
+            placeholder="Title"
+            name="title"
+            className="form__input"
+            id="title"
+            value={title}
+            onChange={(event) => {
+              this.inputHandler(event);
+            }}
```

Comment:
pass just callback if no arguments or the arg is event

```suggestion
            onChange={this.inputHandler}
```

Reply:
+++